addons/l10n_ar/demo/product_product_demo.xml

```diff
@@ -7,6 +7,7 @@
         <field name="default_code">AFIP_DESPACHO</field>
         <field name="sale_ok" eval="False"/>
         <field name="uom_id" ref="uom.product_uom_unit"/>
+        <field name="company_id" ref="company_ri"/>
```

Comment:
Pretty sure these products need to be usable by all the AR demo companies (RI, Mono and Exento)